### PR TITLE
Replace styled button with toolbar button in theme

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR cleans up `filtering` by removing the `styledbutton` and moving that to theme.

For now, `toolbar` has been selected because it conveys information about where this button is typically used. If the usage of this button expands over time, we can always alias it in the theme with a new name but this seems appropriate for the information we have about this type so far.

#### Where should the reviewer start?
aries.js
#### What testing has been done on this PR?
site 
#### How should this be manually tested?
look at deployment
#### What are the relevant issues?
issue #1118 
#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: aries-site/src/examples/templates/filtering/FilteringWithRangeSelector.js [hunk 1]
```diff
@@ -235,7 +220,8 @@ const Filters = ({
     return (
       <>
         <Box align="center" direction="row" gap="xsmall">
-          <StyledButton
+          <Button
+            king="tertiary"
```
Comment (taysea) [inline aries-site/src/examples/templates/filtering/FilteringWithRangeSelector.js:224]:
```suggestion
            kind="tertiary"
```

**Review by taysea — COMMENTED** [review]
Looks good, will wait on discussion for naming before approving.

File: aries-site/src/examples/templates/filtering/FilteringWithRangeSelector.js (hunk 1 above)
Comment (britt6612) [reply aries-site/src/examples/templates/filtering/FilteringWithRangeSelector.js]: thanks for catching :)

File: aries-site/src/examples/templates/filtering/FilteringWithSelect.js [hunk 2]
```diff
@@ -224,7 +217,8 @@ const Filters = ({
         {size !== 'small' ? (
           content
         ) : (
-          <StyledButton
+          <Button
+            king="toolbar"
```
Comment (taysea) [inline aries-site/src/examples/templates/filtering/FilteringWithSelect.js:221]:
```suggestion
            kind="toolbar"
```

Comment (taysea) [inline aries-site/src/examples/templates/filtering/FilteringWithSelect.js]:
```suggestion
            kind="toolbar"
```

**Pull request merged**